src/js/components/FocusedContainer.js

```diff
@@ -44,24 +161,31 @@ export const FocusedContainer = ({
   }, [bodyOverflowStyle, hidden, trapFocus, restrictScroll]);
 
   useEffect(() => {
+    const roots = contextRoots.current;
     const timer = setTimeout(() => {
-      if (!hidden && trapFocus && roots && roots[0]) {
-        roots.forEach(makeNodeUnfocusable);
+      if (!hidden && trapFocus && roots?.[0]) {
```

Comment:
I don't think you need the `roots?.[0]` check, at least if you change line 68 to do `roots?.forEach`

Reply:
You're right, I think this was lingering from old logic.

Reply:
Done.